gordo_components/builder/build_model.py

```diff
@@ -48,10 +49,16 @@ def build_model(
         Mapping of the Dataset to initialize, following the same logic as model_config.
     metadata: dict
         Mapping of arbitrary metadata data.
+    cv_mode: str
+        String which enables three different modes:
+        * cross_val_only: Only printing cv_scores after cross validation, build the model i.e.
+        no cross validation or
+        * build_only: Skips cross validation and only builds the model
```

Comment:
Imperative>descriptive. So `Skip cross validation and only build the model` instead of "Skips cross validation and only builds the model". And similar for the other ones.